docs/UserGuide.md

```diff
@@ -62,111 +74,128 @@ clean Graphical User Interface (GUI) for easy comprehension of expenditure and s
-  `command | identifier | input | [optional inputs]`<br>
-  Example: `add -s 15.60 / 15-08-2022`<br>
+  `command | entry identifier | input | [optional inputs]`<br>
+  Example: `add t/e d/Lunch a/15.60 da/15-08-2022 c/Meal` <br>
     - Command : add<br>
-    - Identifier : -s<br>
-    - Input : 15.60<br>
-    - Optional input : 15-08-2022<br>
+    - Identifier : t/e<br>
+    - Input : d/Lunch a/15.60 da/15-08-2022 c/Meal<br>
+    - Optional input : NIL<br>
 
 - Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `del -s INDEX_OF_SPENDING`, `INDEX_OF_SPENDING` is a parameter which can be used as `del -s 10`.
+  e.g. in `del INDEX_OF_ENTRY t/ENTRY_TYPE`, `INDEX_OF_ENTRY` is a parameter which can be used as `del 10 t/e`.
 
 - Items in square brackets are optional.<br>
-  e.g `add -s NUMBER [DATE]` can be used as `add -s 100 [31-08-2022]` or as `add -s 100`.
+  e.g `view t/ENTRY_TYPE g/GRAPH_TYPE [MONTH]` can be used as `view t/e g/c [mo/2022-05]` or as `view t/e g/c`.
 </div>
 
+## Before Using
 
-### Adding spendings: `add -s  NUMBER [DATE]`
-Adds a spending entry. <br>
-1. Add spending with default date <br>
-   - Example: `add -s 15.60`<br>
-   - A spending of 15.60 will be added with the current date
-
-
-2. Add spending with specific date<br>
-   - Example: add `-s 15.60 / 12-09-2022`<br>
-   - A spending of 15.60 will be added with 12 September 2022 as the date
-
-### Deleting spendings: `del -s INDEX_OF_SPENDING`
-
-Deletes a spending entry.
-
-Example: `del -s 2` deletes the 2nd item on the list
-
-Spending list:
-1. Spent 15.60 12 Sep 2022
-2. Spent 1.20 12 Sep 2022
-
-Expected: `A spending of 1.20 on 12 Sep 2022 has been deleted.`
-
+<div markdown="block" class="alert alert-info">
 
-### Adding income: `add -i NUMBER [DATE]`
+**:information_source: Notes about the application:**<br>
+- For **ALL** entries, categories are <ins>compulsory</ins> and every entry can only contain <ins>one</ins> category. 
+  The following table shows the`Expenditure` and `Income` entries categories.
+
+|   `Expenditure`   |    `Income`     |
+|:-----------------:|:---------------:|
+|     **Food**      |   **Salary**    |
+|   **Groceries**   |  **Allowance**  |
+| **Entertainment** |   **Profit**    |
+|   **Education**   | **Investments** |
+|    **Housing**    |    **Gifts**    |
+|    **Others**     |   **Others**    |
+
+- For **ALL** entries, 2 entries are considered duplicates <ins>IF</ins> both entries have exactly the same:
+  `description`, `date` `amount` and `category`. We do not want PennyWise to be managing entries which are a repeat,
+  of one another, as it would be easier to simply use the [edit](#editing-entries--edit-index_of_entry-tentry_type-dedited_description-aedited_amount-daedited_date-cedited_category) 
+  command to alter the original entry's specifications.
+  e.g. Executing the following 2 commands one after another, PennyWise will recognise (2) as a duplicate entry. 
```

Comment:
would adding a second portion on what is accepted be gd? like maybe varying description or amount